app/src/main/java/com/example/habitsmasher/FollowUserDialog.java

```diff
@@ -156,15 +172,6 @@ private void addNewFollowerForUserInDatabase(String userId, String newFollowerId
         userRef.update(FOLLOWERS_FIELD, FieldValue.arrayUnion(newFollowerId));
     }
 
-    /**
-     * This helper method gets the userID of the current user that is logged in
-     * @return the current user's ID
-     */
-    public String getCurrentUserId() {
-        SharedPreferences sharedPref = getContext().getSharedPreferences(USER_DATA_PREFERENCES_TAG, Context.MODE_PRIVATE);
-        return sharedPref.getString(USER_ID_SHARED_PREF_TAG, "id");
-    }
```

Comment:
Not deleted, this method was a duplicate. A similar method already exists in the `UserDatabaseHelper`!